topics/admin/tutorials/interactive-tools/tutorial.md

```diff
@@ -51,23 +56,22 @@ recordings:
 
 Galaxy Interactive Tools (GxITs) are a method to run containerized tools that are interactive in nature. Interactive Tools typically run a persistent service accessed on a specific port and run until terminated by the user. One common example of such a tool is [Jupyter Notebook][jupyter]. Galaxy Interactive Tools are similar in purpose to [Galaxy Interactive Environments][gie-docs] (GIEs), but are implemented in a significantly different manner. Most notably, instead of directly invoking containers on the Galaxy server, dedicated Docker node, or as a Docker Swarm service (as is done for GIEs), Interactive Tools are submitted through Galaxy's job management system and thus are scheduled the same as any other Galaxy tool - on a Slurm cluster, for instance. Galaxy Interactive Tools were introduced in Galaxy Release 19.09.
 
-> <warning-title>Evolving Topic, Not Updated for 23.0 + Gravity</warning-title>
-> Galaxy Interactive Tools are a **new feature** and there are some rough edges. Work to improve the experience of deploying and using them is ongoing.
-{: .warning}
+> <comment-title>Evolving Topic</comment-title>
+> Galaxy Interactive Tools are a **relatively new and rapidly evolving feature** and there are some rough edges. Work to improve the experience of deploying and using them is ongoing. Please watch the [Galaxy Release Notes][galaxy-release-notes] for updates, changes, new documentation, and bug fixes. 
+>
+> This tutorial has not been updated for Galaxy 23.0+ and Gravity. You may find extra information about Interactive Tools on the [Galaxy Documentation][galaxy-docs-interactivetools].
+{: .comment}
+
+[galaxy-release-notes]: https://docs.galaxyproject.org/en/master/releases/index.html
+[galaxy-docs-interactivetools]: https://docs.galaxyproject.org/en/master/admin/special_topics/interactivetools.html
 
-> <warning-title>Before You Continue</warning-title>
+> <warning-title>Before You Continue - Wildcard DNS Certificates</warning-title>
 > If you are *not* completing this tutorial as part of a [Galaxy Admin Training][gat] course, **you will need a wildcard DNS record for your Galaxy server and a method for obtaining a wildcard SSL certificate for your Galaxy server**.
```

Comment:
do all (or at least, most) of the interactive tools work with path-based proxying?

if so, I think it is worth it to focus on the path-based proxying primarily / only. It would remove a significant amount of very complicated setup, and it also just simply isn't available for most galaxy deployments in educational institutions.

Reply:
No, quite many need a subdomain. A few examples: [geoexplorer](https://github.com/galaxyproject/galaxy/blob/d3fb3fdf7b8ff898729b4ed3814a2a73fa77b872/tools/interactive/interactivetool_geoexplorer.xml#L7), [radiant](https://github.com/galaxyproject/galaxy/blob/d3fb3fdf7b8ff898729b4ed3814a2a73fa77b872/tools/interactive/interactivetool_radiant.xml#L7), [HiCBrowser](https://github.com/galaxyproject/galaxy/blob/d3fb3fdf7b8ff898729b4ed3814a2a73fa77b872/tools/interactive/interactivetool_hicbrowser.xml#L6), [HiGlass](https://github.com/galaxyproject/galaxy/blob/d3fb3fdf7b8ff898729b4ed3814a2a73fa77b872/tools/interactive/interactivetool_higlass.xml#L7), [iSEE](https://github.com/galaxyproject/galaxy/blob/d3fb3fdf7b8ff898729b4ed3814a2a73fa77b872/tools/interactive/interactivetool_isee.xml#L7). These instructions should stay imo.